src/python/platforms/fuchsia/util/device.py

```diff
@@ -253,17 +253,24 @@ def process_logs(self, logfile, guess_pid=False):
     mutation_pattern = re.compile(r'^MS: [0-9]*')
     artifacts = []
     artifact_pattern = re.compile(r'Test unit written to data/(\S*)')
+    repro_pattern = re.compile(r'Running: .*')
+    line_of_actual_crash = None
     with open(logfile) as log:
       with open(logfile + '.tmp', 'w') as tmp:
         for line in log:
           # Check for a line that tells us the process ID
           match = pid_pattern.search(line)
           if match:
+            line_with_crash_message = line
```

Comment:
this hack is fine for now, but can this ordering be fixed on the device side?

Reply:
yeah, the plan is to patch up compiler-rt so that Fuchsia emits things the proper way in the first place.  i added a TODO a few lines down so this isn't lost, but rest assured we're working on a real fix!